### ISSUE TITLE
Texte vom Zeltlager überarbeiten

### ISSUE THREAD

**Issue by hmoldenhauer** [kjg-dortmund-wickede/kjg-dortmund-wickede.github.io#14]
Wir sollten ein gemeinsames Format für die Texte des Lagers finden.
Der Text von '99 ist okay, 2000 - 2002 gefallen mir nicht.

**Comment by tuerex** [comment]
Ja, ich bin auch deiner Meinung. Ich hatte einfach nur die Inhalte von der alten Website kopiert um ein Gefühl dafür zu bekommen wieviel Informationen da hin sollen. Vielleicht reicht der Ort für die Vergangenheit und ein bisschen mehr für die Zukunft.
Evtl hat Berni die Zeitungsartikel von den Jahren? Das könnte eine gute Information sein.
